cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -231,6 +231,39 @@ if(builtin_lzma)
 endif()
 
 
+#---Check for LZ4--------------------------------------------------------------------
+if(NOT builtin_lz4)
```

Comment:
Ah, I see. That part is tricky to read. Maybe we could add a clarification comment for the casual reader.